app-admin/gentoo-perl-helpers/gentoo-perl-helpers-0.4.0.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=8
 
 DESCRIPTION="Introspection and Upgrade Assistance tools for Gentoo Perl"
 HOMEPAGE="https://github.com/gentoo-perl/gentoo-perl-helpers"
```

Comment:
* Drop the silly hack in RDEPEND and just put a proper comment above it. (If nothing else, completely unnecessary subshell in global scope.)
* Also, change DEPEND to BDEPEND.
* Drop blank IUSE.
* Add a blank new line between phase definitions.